### PR TITLE
Updating workflows

### PR DESCRIPTION
**Pull request by jenniferboedker**

### CODE REVIEW

File: .github/release.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+    authors:
+      - JohnnyQ5
```
Comment (Steffengreiner) [inline .github/release.yml:6]: See newest template release 
```suggestion
      - JohnnyQ5
      - github-actions
```

**Review by Steffengreiner — CHANGES_REQUESTED** [review]
Very nice 👨‍🎤, however you have to adapt the workflows to the current template: 
1.) Remove duplicate release.yml 
2.) Remove groovy-checkstyle.yml 
3.) Adapt nexus-publish.yml to nexus-publish-snapshot.yml.

**Comment by jenniferboedker** [comment]
@Steffengreiner heya, thanks for the feedback, I would argue that the checkstyle can stay, won't hurt anyone. But I addressed the other changes

**Comment by jenniferboedker** [comment]
see #220 with correct branchname

**Pull request closed without merging**